tests/test_card.py

```diff
@@ -195,47 +195,27 @@ def test_sets(client):
     response = client.get("/card/The%20Line")
```

Comment:
So many changes here that have no impact... I don't see what is the actual change in it